djangosaml2/views.py

```diff
@@ -572,7 +573,15 @@ def post(self, request, attribute_mapping=None, create_unknown_user=None):
         custom_redirect_url = self.custom_redirect(user, relay_state, session_info)
         if custom_redirect_url:
             return HttpResponseRedirect(custom_redirect_url)
+
         relay_state = validate_referral_url(request, relay_state)
+        if relay_state is None:
```

Comment:
```suggestion
        if not relay_state:
```

it is more flexible in the cases where the relaystate is a void string ""